src/components/MoviesMenu/Movies/Movies.module.scss

```diff
@@ -0,0 +1,32 @@
+@import "styles/theme";
+.moviesLayout {
+  margin-top: 20px;
+}
+
+.moviesList {
+  list-style: none;
+  display: $flexedItem;
```

Comment:
`$flexedItem` is an unnecessary abstraction. It just creates an alias for an existing CSS property. If you have a lot of flex styles you're going to re-use, I'd suggest you combine them into named classes that you can extend. For example, if you have a lot of sections that are styled flex, space between, center, you could do something like the following in your `theme.scss`:

```scss
.horizontalCenterBetween {
  display: flex;
  justify-content: space-between;
  align-items: center;
}
```

Then you can extend this, as follows, in another SCSS file:

```scss
.moviesList {
  @extend .horizontalCenterBetween;
  // ... rest of your styles
}
```